### ISSUE TITLE
Update talks about version 3.9, but conda does not have that?

### ISSUE THREAD

**Issue by dutchscientist** [ncbi/amr#42]
When I do amrfinder -u, it says that the Nov 2020 database requires updating to version 3.9. But that is not in the version overview here, nor is it in Conda.

Furthermore, the instructions on the update page say "conda update amrfinderplus", but this should be "conda update ncbi-amrfinderplus"?

**Comment by evolarjun** [comment]
Sorry, you caught us in the middle of a release of a new version of software and database with a change in the format. The new release is up here (https://github.com/ncbi/amr/releases/tag/amrfinder_v3.9.3), and hopefully bioconda will be updated soon. It takes a little time for changes to propagate through there. You can use the binary or source options if you need the latest database version, but the previous software and database versions should continue to work in the meantime.

If you are having problems getting AMRFinderPlus to run please let us know.

**Comment by dutchscientist** [comment]
thanks and no worries. Just my timing ;-)

As a note: on https://github.com/ncbi/amr/wiki/Upgrading it has the incorrect syntax for the update ("conda update amrfinderplus" instead of "conda update **ncbi**-amrfinderplus".

Will try again in a few days. AMRfinder is an awesome tool!

**Comment by evolarjun** [comment]
Thank you so much for noticing the typo! I just checked and tests are running right now on bioconda, so version 3.9.3 should be out and propagate to the servers by sometime tomorrow.

**Comment by evolarjun** [comment]
Version 3.9.3 is available in bioconda. I haven't had any trouble installing it myself, but my automated conda installation tests are struggling, so I'm nervous.  Were you able to install?

**Comment by dutchscientist** [comment]
It updated fine on most of my machines once I made sure the base environment was activated in Conda (on some training machines which are only command line it wasn't). I had two virtualbox computers where it had previously not updated from 3.8.4, which I hadn't noticed, and those refused to just do it with my "conda update --all --yes".

I had to specify "conda install ncbi-amrfinderplus==3.9.3" with those two, took them a while but then they updated fine.

So any problems mostly were with settings on my machines?
